src/langchain_google_cloud_sql_pg/postgresql_loader.py

```diff
@@ -167,16 +166,15 @@ async def create(
         if format and formatter:
             raise ValueError("Only one of 'format' or 'formatter' should be specified.")
 
-        if format and format.lower() not in ["csv", "text", "json", "yaml"]:
+        if format and format not in ["csv", "text", "JSON", "YAML"]:
             raise ValueError("format must be type: 'csv', 'text', 'JSON', 'YAML'")
-        format = format.lower() if format else format
         if formatter:
             formatter = formatter
         elif format == "csv":
             formatter = cvs_formatter
```

Comment:
nit: typo I believe, should be `csv_formatter` i believe. Probably needs to be updated multiple places...
```suggestion
            formatter = csv_formatter
```